### ISSUE TITLE
CurrentDetailsOnMap crashing when no places returned

### ISSUE THREAD

**Issue by WayneRiesterer** [googlemaps/android-samples#62]
Hello :)

If no places are returned, the app crashes when trying to show the dialog:

```
        // Display the dialog.
        AlertDialog dialog = new AlertDialog.Builder(this)
                .setTitle(R.string.pick_place)
                .setItems(mLikelyPlaceNames, listener)
                .show();
```
This is because `mLikelyPlaceNames == null`.

**Comment by opsinghis** [comment]
The throws an exception which is causing the application in the emulator to crash. 
```
mLastKnownLocation = task.getResult();
                            mMap.moveCamera(CameraUpdateFactory.newLatLngZoom(
                                new LatLng(mLastKnownLocation.getLatitude(),
                                      mLastKnownLocation.getLongitude()), DEFAULT_ZOOM));
```

**Comment by wangela** [comment]
Fixed with null check in PR #141
